support-modules/acquisition-events/src/main/scala/com/gu/support/acquisitions/ga/GoogleAnalyticsServiceImpl.scala

```diff
@@ -0,0 +1,255 @@
+package com.gu.support.acquisitions.ga
```

Comment:
This file is similar to the original GAService, but it receives the scala acquisitions model instead of thrift